lib/carbonite/prefix.ex

```diff
@@ -0,0 +1,12 @@
+defmodule Carbonite.Prefix do
+  @moduledoc false
+
+  @doc false
```

Comment:
```suggestion
```

`@moduledoc false` should be enough, no?

Nice extraction by the way. Wonder if we should call the module `Carbonite.Schema` instead?